backend/src/main/java/com/rikishi/backend/matching/systems/TreeBracket.java

```diff
@@ -39,16 +44,51 @@ public Set<Player> getCurrentPlayers() {
 
     @Override
     public void loadPlayers(Collection<Player> players) {
+        List<Player> playersArray = new ArrayList<>(players);
```

Comment:
można np. zapisać to krócej `var playersArray = new ArrayList<Player>(players)` - ale nie ma wielkiej różnicy